src/conversion/model-to-view-converters.js

```diff
@@ -492,21 +492,24 @@ export function rename() {
 }
 
 /**
- * Function factory, creates a default converter for inserting {@link module:engine/model/item~Item model item} into a marker range.
+ * Function factory, creates a default converter for inserting {@link module:engine/model/item~Item model item} into a marker range
+ * or inserting a part of a model that contains a marker (this happens when reinserting nodes from graveyard).
  *
  *		modelDispatcher.on( 'insert', insertIntoRange( modelDocument.markers ) );
  *
  * @param {module:engine/model/markercollection~MarkerCollection} markerCollection Markers collection to check when
  * inserting.
  * @returns {Function}
  */
-export function insertIntoMarker( markerCollection ) {
+export function insertMarkerOrIntoMarker( markerCollection ) {
```

Comment:
Robert C. Martin told once that if you have a conjunction in your function name you are doing something wrong.